Sources/Features/CreateAccount/Coordinator/CreateAccountCoordinator+Models.swift

```diff
@@ -41,7 +41,7 @@ extension CreateAccountConfig {
 		case let .firstAccountOnNewNetwork(specificNetworkID):
 			self.init(
 				isFirstAccount: true,
-				canBeDismissed: false,
+				canBeDismissed: true,
```

Comment:
Should this really be dismissible? I mean it is mandatory to create an account on network,  this allows bypassing that.

Reply:
It can still be dismissed, because it's a sheet in this case, so you can slide it down. But if you do that, or press the close button in the navbar that this flag enables, then you just go back to the gateway selection screen.

Reply:
ooh, ok, I thought this `canBeDismissed` is also used to disable the dismissal of the sheet by sliding it down by setting `.interactiveDismissDisabled` on the sheet.